apis/Google.Cloud.Diagnostics.Common/Google.Cloud.Diagnostics.Common/FlushableConsumerBase.cs

```diff
@@ -150,14 +151,14 @@ private void Flush(IEnumerable<T> items)
         /// <param name="cancellationToken">The token to monitor for cancellation requests.</param>
         /// <param name="items">The items to flush.</param>
         /// <returns>A task representing the asynchronous operation.</returns>
-        private Task FlushAsync(IEnumerable<T> items, CancellationToken cancellationToken = default)
+        private async Task FlushAsync(IEnumerable<T> items, CancellationToken cancellationToken = default)
```

Comment:
I don't see why we've got this change. Awaiting a completed task won't do anything - you could just return. So why make the method async?